pkg/expect/http.go

```diff
@@ -143,6 +143,11 @@ func (exp *AppExpectation) createDataStoreHTTP(id uuid.UUID) *config.DatastoreCo
 			ds.DType = config.DsType_DsHttps
 		}
 		ds.Fqdn = fmt.Sprintf("%s://%s", u.Scheme, u.Hostname())
+		// In case the http datastore address has a port number,
+		// the datastore address should contain both the hostname and the port.
+		if u.Port() != "" {
+			ds.Fqdn = fmt.Sprintf("%s:%s", ds.Fqdn, u.Port())
```

Comment:
Can we use `u.Host` instead in both cases (as full host:port notation), looks like it contains the port and `u.Port()` just extracts it from `u.Host`.

Reply:
Sure.